lib/sablon/image.rb

```diff
@@ -10,7 +10,8 @@ def inspect
     end
 
     def self.create_by_path(path)
-      Sablon::Image::Definition.new(Pathname.new(path).basename.to_s, IO.binread(path))
+      image_name = "#{Random.new_seed}-#{Pathname.new(path).basename.to_s}"
```

Comment:
Is this causing the comparison to the fixture to fail due to the random nature of the file name?